src/dynarec/rv64/dynarec_rv64_66f0.c

```diff
@@ -49,72 +49,7 @@ uintptr_t dynarec64_66F0(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
 
     GETREX();
 
-    switch(opcode) {
-        case 0x11:
-            INST_NAME("LOCK ADC Ew, Gw");
```

Comment:
Note that I deleted this opcode because I found that the implementation wrong, I'll add it back later.